print ("Handling exception using try...except...else...finally")
try:
    numerator=50
    denom=int(input("Enter the denominator: "))
    quotient=(numerator/denom)
    print ("Division performed successfully")
except ZeroDivisionError:
    print ("Denominator as ZERO is not allowed")
#except ValueError:
 #   print ("Only INTEGERS should be entered")
except Exception:
    print("exception occur")
else:
    print ("The result of division operation is ", quotient)
finally:
    print ("OVER AND OUT..........") 
